# goal
int8 induct_station
int16 drop
int32 induct_x
int32 induct_y
int32 goal_x
int32 goal_y
string city
string pkg_id
---
# result
bool flag_success
---
# feedback
int8 percentage_complete
